workspace-manager-clienttests/src/main/resources/configs/integration/BasicAuthenticated.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
why is integration under client tests? I thought those were an independent category. Now `GetWorkspace` is in two config files. Is that intentional?

Reply:
The following updates were made after following up with @marikomedlock.

We will keep to the convention of having separate subfolders under configs/ to group tests under different test categories. 

For example, under the `configs/` folder, there are two subfolders: `integration` and `perf`.
Inside each subfolder (`integration` or `perf`), we have the test configuration files. 
The test configuration files determine the target scenario, user load, which server config to use etc.
So even the same scenario `GetWorkspace` was used, the target load and server config can be different for `integration` vs `perf`. 

Also the `suites/` folder is important, the `json` files inside this folder defines how the tests are grouped together. The sample GitHub action workflows `test-runner-nightly-perf.yml`, `test-runner-integration.yml` use the `json` definitions in `suites/` folder to launch the appropriate tests.

Hope this makes it all clear.

Reply:
Is there ever a reason to have more than one test script in a test config file (as opposed to putting them in suites)?

Reply:
Yes, there is. A test script implements a particular user journey. A test config specifies which user journeys to run and how to scale them. A test config can run several different user journeys concurrently to test a varying load (e.g. we want to test that the server survives 100 workspace create requests and 200 workspace get requests at the same time). Let me know if it's not clear -- happy to talk on a call also.